### ISSUE TITLE
Last line and bottom of scroll bar are cut off on all generated web pages

### ISSUE THREAD

**Issue by lyon-fnal** [KosmosisDire/obsidian-webpage-export#130]
**Describe the bug**
The last line and the bottom of the scroll bar of all generated web pages are cut off (see screen shot below). This happens in both Safari and Chrome. This happens for both the default and Minimal themes (those are the two that I tried). If I add extra whitespace to the bottom of the markdown page (e.g. blank lines), that whitespace is not present in the generated web page, so the last line of text still gets cut off. 

I have no extra CSS active (e.g. from the Appearance setting).

**Operating System**
MacOS Ventura 13.4 .

**To Reproduce**
Export any page (or multiple pages). The cut off always happens on every generated web page. 

**Screenshots**
![bottom](https://github.com/KosmosisDire/obsidian-webpage-export/assets/9458395/b7b58bb3-6390-4321-b25a-2e14e15c0ddb)

You see the last line is cut off as is the scroll bar (bottom should be rounded). 

### IMPORTANT: Error Log
No errors

**Comment by lyon-fnal** [comment]
Well, for what it's worth, I can make the last line not cut off by changing the `.webpage-container` settings within `plugin-styles.css` ...

From
```css
.webpage-container {
    background-color: var(--background-primary);
    display: flex;
    flex-direction: row;
    height: 100%;
    width: 100%;
    align-items: stretch;
    position: fixed;
}
```

to 
```css
.webpage-container {
    background-color: var(--background-primary);
    display: flex;
    flex-direction: row;
    height: 99%;     /* Change this line */
    width: 100%;
    align-items: stretch;
    position: fixed;
}
```

The bottom of the scroll bar is still cut off, but the last text line shows fully.

**Comment by lyon-fnal** [comment]
I should have started this thread with a big thank you for writing this plugin! It's really great!

**Comment by lyon-fnal** [comment]
Awesome! I'll await the next release. Thanks!

**Comment by KosmosisDire** [comment]
This is fixed in v1.7.1
